test/redeyed-result.js

```diff
@@ -18,7 +18,7 @@ test('redeyed result has esprima ast, tokens, comments and splits and transforme
     , tokens =  ast.tokens
     , comments = ast.comments
 
-    , result = redeyed(code, conf)
+    , result = redeyed(code, conf, { buildAst: true } )
```

Comment:
Probably should add a test here with default opts. Asserting that it has no `ast` but everything else.